flake8_quotes/__init__.py

```diff
@@ -12,31 +12,51 @@ class QuoteChecker(object):
     QUOTES = {
```

Comment:
Maybe we should rename this to `INLINE_QUOTES` as well as the other references to avoid ambiguity?